research.md

```diff
@@ -0,0 +1,18 @@
+Currently existing python libraries:
+
+ESpeak
+Python speech-synthesizer program which accepts text input and generates an audio file (either mp3 or wav). Multiple different voice synthesizer options.
+Djangulo go-espeak is a potential alternative: https://pkg.go.dev/github.com/djangulo/go-espeak#section-readme
```

Comment:
I believe eSpeak-NG would be better than the standard eSpeak.  There is a separate wrapper:
https://github.com/BenLubar/espeak